app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitDocumentsApiController.php

```diff
@@ -343,4 +345,49 @@ public function removeEventType($summit_id, $document_id, $event_type_id){
         }
     }
 
+    /**
+     * @param LaravelRequest $request
+     * @param $summit_id
+     * @param $document_id
+     * @return \Illuminate\Http\JsonResponse|mixed
+     */
+    public function addFile(LaravelRequest $request, $summit_id, $document_id){
+        return $this->processRequest(function () use ($request, $summit_id, $document_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $file = $request->file('file');
+            if (is_null($file)) {
+                return $this->error412(array('file param not set!'));
+            }
+
+            $document = $this->service->addFile2SummitDocument($summit, intval($document_id), $file);
+
+            return $this->updated(SerializerRegistry::getInstance()->getSerializer($document)->serialize(
+                SerializerUtils::getExpand(),
+                SerializerUtils::getFields(),
+                SerializerUtils::getRelations()
+            ));
+        });
+    }
+
+    /**
+     * @param $summit_id
+     * @param $document_id
+     * @return \Illuminate\Http\JsonResponse|mixed
+     */
+    public function removeFile($summit_id, $document_id){
+        return $this->processRequest(function () use ($summit_id, $document_id) {
+            $summit = SummitFinderStrategyFactory::build($this->summit_repository, $this->resource_server_context)->find($summit_id);
+            if (is_null($summit)) return $this->error404();
+
+            $document = $this->service->removeFileFromSummitDocument($summit, intval($document_id));
+
+            return $this->deleted(SerializerRegistry::getInstance()->getSerializer($document)->serialize(
```

Comment:
delete dont need to return anything, generally we return 204 without content
check another similar places